### PR TITLE
feat(web-console): Fix the response to handle all case of table name

### PR DESCRIPTION
**Pull request by AayushSaini101**
Resolves: #324 
* Allow the web console to handle all cases of table names.
* Example: Table Name student and Student must be treated the same.

### CODE REVIEW

**Comment by AayushSaini101** [comment]
@glasstiger can you please review this ?

**Comment by glasstiger** [comment]
@AayushSaini101 change looks good but would be nice to add tests.
I think easiest is to create a cypress test which replicates the issue reported.

**Comment by AayushSaini101** [comment]
@glasstiger Added thanks

File: packages/browser-tests/cypress/integration/console/schema.spec.js [hunk 1]
```diff
@@ -18,6 +18,19 @@ describe("questdb schema with working tables", () => {
     });
     cy.refreshSchema();
   });
+  
+  it("should throw a warning when table is present with different case",()=>{
```
Comment (glasstiger) [inline packages/browser-tests/cypress/integration/console/schema.spec.js:22]: Hey @AayushSaini101 , it looks to me that this test does not test your change.
This test goes to the backend to create the table.
What you would need to do is use cypress to open the table schema dialog, type in "Btc_Trades", and assert that the error message appeared.

I do not think you need to create any tables on the backend.
It looks to me that the 4 tables are created before every test in a `before()` hook, so they should exist already.
No need for this code:
```
    tables.forEach((table) => {
      cy.createTable(table);
    });
```

Check out this test in this file at line 43:
```
it("should filter the table with input field", () => {
  ...
}
```

Not exactly what you need, but it is a good example how to find elements on the UI with cypress, and how to simulate typing in text into an input, for example.

Comment (AayushSaini101) [reply packages/browser-tests/cypress/integration/console/schema.spec.js:22]: Hope this works thanks @glasstiger 
```
  it("should throw a warning when table is present with different case",()=>{
    cy.get('input[name="table_filter"]').type("Btc_trades").should("contains","Table name must be unique")
    cy.get('input[name="table_filter"]').type("gitLog").should("contains","Table name must be unique")
    cy.get('input[name="table_filter"]').type("GITLOG").should("contains","Table name must be unique")
  })
```

Comment (glasstiger) [reply packages/browser-tests/cypress/integration/console/schema.spec.js:22]: Hi @AayushSaini101 , is this test passing for you when you run it?

Comment (glasstiger) [reply packages/browser-tests/cypress/integration/console/schema.spec.js:22]: Please, remove the test.
We can merge it.
It is a tiny change.

Comment (AayushSaini101) [reply packages/browser-tests/cypress/integration/console/schema.spec.js]: Done @glasstiger Please merge

**Review by glasstiger — APPROVED** [review]

**Pull request merged**